Misc/NEWS.d/next/Core_and_Builtins/2024-09-25-11-53-22.gh-issue-124442.EXC1Ve.rst

```diff
@@ -0,0 +1,2 @@
+Make the :func:`compile` builtin function deterministic by sorting
```

Comment:
```suggestion
Fix nondeterminism in compilation by sorting
```
It's not just `compile()` but also general bytecode compilation. Also, there may be other sources of nondeterminism.

Reply:
Fixed

Reply:
The news entry now doesn't read right as you have kept the first line but deleted the second. In the future, an easy way to accept suggestions like this is to click the 'commit' button in the GitHub interface.

A